app/livechat/client/views/app/tabbar/customerChatHistory.js

```diff
@@ -0,0 +1,230 @@
+import { ReactiveVar } from 'meteor/reactive-var';
+import { Template } from 'meteor/templating';
+import moment from 'moment';
+import _ from 'underscore';
+import './customerChatHistory.html';
+import { APIClient, t } from '../../../../../utils/client';
+import { AccountBox, TabBar, MessageTypes } from '../../../../../ui-utils';
+import {Mongo ,MongoInternals} from 'meteor/mongo';
+import { Tracker } from 'meteor/tracker';
+import { ReactiveDict } from 'meteor/reactive-dict';
+import { Session } from 'meteor/session';
+
+const ITEMS_COUNT = 50;
+let  historyResult 
+let len, msgs
+let allHistoryChat ;
+
+Template.customerChatHistory.helpers({
+	
+	isAllChat:function(){
+        // will return is have to load all chat 
+		return Template.instance().isAllChat.get();
+	},
+	isChatClicked:function(){
+		// will return that if you have clicked in a single chatHistory
+		return Template.instance().isChatClicked.get();
+	},
+	isfound:function() {
+		// will return if find any search result
+		var isfound= Session.get('found');
+	
+		return isfound;
+	},
+	isLoading() {
+		return Template.instance().isLoading.get();
+	},
+	searchResults(){
+		// will return search result
+		var r = Session.get('searchResult');
+		return 	Template.instance().searchResult.get();
+	},
+	
+	previousChats() {
+		// will return pervious chats list
+		let history = Template.instance().history.get();
+		let newHisTory = []
```

Comment:
You don't need any of this code between the lines 46 ~ 58.

Reply:
i need this because of i am adding a time property inside the message object then pushing into a new array